index.html

```diff
@@ -582,9 +582,10 @@ <h3>Processing model</h3>
    <dd>"<code>no-cache</code>"
   </dl>
 
- <li><p>Let <var>response</var>’s <a>body</a> be a JSON <a>Object</a>
-  with a key "<code>value</code>"
-  set to the <a>JSON serialization</a> of <var>data</var>.
+ <li><p>Let <var>response</var>’s <a>body</a> be the UTF-8
+  encoded <a>JSON serialization</a> of a JSON <a>Object</a> with a key
```

Comment:
We should reference [UTF-8 encode](https://encoding.spec.whatwg.org/#utf-8-encode) from INFRA.

Reply:
I shall add this.